### DISCUSSION TITLE
Configuration

### DISCUSSION THREAD

**Discussion by markuswesterberg** [plmilord/Hass.io-custom-component-spaclient#39] [General]
So when adding the integration I need to provide "Host" and "Name". I assume host is the IP-address of the SPA on my network? What would be the name?

**Comment by plmilord** [comment]
Host = IP address of the balboa module on your network
Name = simply name your spa whatever you want!

P-L

**Reply by markuswesterberg** [reply]
OK, thanks. But I get error message "failed to connect". Could it be that I already have the Balboa Spa Client integration (another integration) set up and connected you think? Do I have to remove it or is it possible to have multiple integrations connected to the same device?

**Comment by plmilord** [comment]
Of course, you need to remove the other integration.

The Balboa device can only handle one uplink.
